dm/pkg/schema/tracker.go

```diff
@@ -370,6 +370,22 @@ func (tr *Tracker) CreateTableIfNotExists(table *filter.Table, ti *model.TableIn
 	return tr.dom.DDL().CreateTableWithInfo(tr.se, schemaName, ti, ddl.OnExistIgnore)
 }
 
+func (tr *Tracker) BatchCreateTableIfNotExist(tablesToCreate map[string]map[string]*model.TableInfo) error {
+	for schema, tableNameInfo := range tablesToCreate {
+		var cloneTis []*model.TableInfo
```

Comment:
```suggestion
		var cloneTIs []*model.TableInfo
```

TableInfo consists of two words, so I prefer each words has a capitalized latter

Reply:
Indeed. But in `tracker.go`, `ti` is everywhere to represent the `TableInfo`. I thought it was some sort of convention😂